tests/clickhouse-test

```diff
@@ -1484,7 +1484,7 @@ class TestCase:
                     body = read_file_as_binary_string(file_path)
                     clickhouse_execute(
                         args,
-                        f"INSERT INTO system.coverage_log SELECT now(), '{self.case}', groupArray(data) FROM input('data UInt64') FORMAT RowBinary",
+                        f"INSERT INTO system.coverage_log SETTINGS async_insert=1, wait_for_async_insert=0 SELECT now(), '{self.case}', groupArray(data) FROM input('data UInt64') FORMAT RowBinary",
```

Comment:
The fixes for tests worked, but we still have too many parts. I need to check what files are we pushing and why and whether we'd need to increase the timings or do something else to support this

Reply:
It was failing in the previous request (uncaught exception) so I've added a try-catch and also increase the `busy_timeout` values